src/commands/stac-validate/__test__/stac.validate.test.ts

```diff
@@ -60,4 +72,80 @@ describe('stacValidate', function () {
     );
     assert.equal(normaliseHref('./item.json', 'collection.json'), 'item.json');
   });
+
+  it('validateStacChecksum', async () => {
+    const memory = new FsMemory();
+    fsa.register('memory://', memory);
+    const path = 'memory://stac/';
```

Comment:
All these tests seem to have this as their first lines and `memory.files.clear();` as their last line, so it would probably make sense to pull them out into `beforeEach`/`afterEach` functions. It might be necessary to pull the tests into a separate `describe` to make sure those functions only run for the relevant tests, though.